resources/js/alpine/toggleAchievementRowsComponent/toggleAchievementRowsComponent.ts

```diff
@@ -0,0 +1,25 @@
+function toggleUnlockedRows() {
+  const allFoundRows = document.querySelectorAll<HTMLLIElement>('li.unlocked-row');
+
+  allFoundRows.forEach((rowEl) => {
+    if (rowEl.classList.contains('hidden')) {
+      rowEl.classList.remove('hidden');
+    } else {
+      rowEl.classList.add('hidden');
+    }
+  });
+}
+
+function toggleNonMissableRows(): void {
+  const allRows = document.querySelectorAll<HTMLLIElement>('#set-achievements-list > li');
+
+  allRows.forEach((row) => {
+    if (!row.classList.contains('missable-row')) {
+      row.classList.toggle('hidden');
+    }
+  });
+}
```

Comment:
There's some weird interactions when toggling both states: i.e. if I hide unlocked achievements, then click the missables link, I get a list of all missables, not just locked ones - and the hide unlocked achievements button remains selected. Similarly, if I select missables first, then hide unlocked, I think it's showing me all achievements

Reply:
In latest, it is no longer possible to enable both states.

If state A is enabled and the user attempts to enable state B, then state A will be forcefully disabled.